test.js

```diff
@@ -100,4 +100,33 @@ describe('gce-images', function () {
       });
     });
   });
-});
\ No newline at end of file
+
+  describe('_sortNewestFirst', function () {
+    describe('when imageA.creationTimestamp is newer than imageB.creationTimestamp', function () {
+      it('should return 1', function () {
```

Comment:
But in `getLatest` (https://github.com/stephenplusplus/gce-images/blob/master/index.js#L131), `pop` is being used, so we want the "newest / latest" element as the last element of the sort.